helpers/setup.sh

```diff
@@ -178,7 +178,7 @@ gcloud projects add-iam-policy-binding "${PROJECT_ID}" \
 
 gcloud projects add-iam-policy-binding "${PROJECT_ID}" \
     --member="serviceAccount:${SERVICE_ACCOUNT_EMAIL}" \
-    --role="roles/compute.networkViewer" \
+    --role="roles/compute.networkAdmin" \
```

Comment:
Same comment as above in the setup script.

Reply:
Should also put a comment here explaining the purpose of this role, since this is an admin role that gives more permissions.